src/nimiSlides.nim

```diff
@@ -31,13 +31,28 @@ type
   SlidesTheme* = enum
     Black, Beige, Blood, Dracula, League, Moon, Night, Serif, Simple, Sky, Solarized, White
 
+  SlideTransition* {.pure.} = enum
```

Comment:
Do we want the enum to be pure, i.e. that you have to write `SlideTransition.convex`?

Reply:
My understanding of the {.pure.} pragma was the opposite. I wanted to avoid calling the full name and just use 'convex.' I tried to compile my test code without the pragma, and it wouldn't compile.

Reply:
Huh, that's weird 🤔 

According to the manual it should be the opposite:
> An enum type can be marked as pure. Then access of its fields always requires full qualification.

I'll try running your code without it and see if I can find something

Reply:
I am also getting an error without it so something fishy is going on. But as it seems to work, we can keep the pure pragma for now :)

Reply:
It doesn't even work when doing `SlideTransition.convex` without the pure pragma, really strange 🤯

Reply:
> Huh, that's weird 🤔
> 
> According to the manual it should be the opposite:
> 
> > An enum type can be marked as pure. Then access of its fields always requires full qualification.
> 
> I'll try running your code without it and see if I can find something

I see, the manual is a little bit confusing to me. I tried to understand the pragma based on this example.
https://nim-by-example.github.io/types/enums/

My guess was that the {.pure.} pragma does two things.
1. It forces you to be explicit when there are overlapping names among all the declared enums.
2. It allows you to refer to enums without qualification if there is no ambiguity.